staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -135,6 +135,16 @@ func Unlogged(req *http.Request, w http.ResponseWriter) http.ResponseWriter {
 	return w
 }
 
+// UpdateStackTracePred allows you to change the default predicate that is
+// used to decide whether to put a stack trace of the calling goroutine into the log
```

Comment:
where is that stored? I only see a temporary rl below.

Reply:
`rl` is a pointer to what is stored in the context. We are changing the original object.

Reply:
haven't found an easy way to update `req` with `req.WithContext` and pass it further down the chain.

Reply:
DisableStackTraceForRequest ?

Reply:
ok, ic, we could do that and drop https://github.com/openshift/kubernetes/pull/494/files#r556379143